tools/test-cli-fatjar-version/src/main/java/com/microsoft/playwright/testclifatjarversion/TestApp.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.playwright.testclifatjarversion;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.util.Collections;
+
+public class TestApp {
+  public static void main(String[] args) throws IOException, InterruptedException, URISyntaxException {
+    URI uri = new URI("jar:file:/home/user/.m2/repository/com/microsoft/playwright/driver-bundle/1.28.0/driver-bundle-1.28.0.jar!/driver/linux");
+    FileSystem fs = FileSystems.newFileSystem(uri, Collections.emptyMap());
+    assert (fs != null);
```

Comment:
Let's throw a RuntimeError instead this way it won't depend on whether assertions are enabled or not (I believe they are not enabled in the current CI config).